src/app/utilities/api-clients/user.js

```diff
@@ -94,8 +94,8 @@ export default class user {
     static createNewUser(body) {
         return http.post(`/users`, body);
     }
-    // TODO: new auth work
-    static getUser(id) {
+
+    static getUserEmail(id) {
```

Comment:
I might be wrong but doesn't this still return the user object, rather than the email? The previous name might be more descriptive